docker/Dockerfile

```diff
@@ -1,62 +1,79 @@
-ARG BASE=22.04
-ARG BASE_IMG=nvcr.io/nvidia/tensorrt:${BASE}-py3
+# Base image starts with CUDA
+ARG BASE_IMG=nvidia/cuda:11.7.1-devel-ubuntu18.04
```

Comment:
Can we make this either 22.04 or 20.04

Reply:
We can but I can't test it with my system currently